lib/tests/2wp.js

```diff
@@ -175,6 +179,28 @@ const execute = (description, getRskHost) => {
             expect(batchPegoutCreatedEvent).to.not.be.null;
             expect(batchPegoutCreatedEvent.arguments.releaseRskTxHashes.includes(pegoutTransaction.transactionHash)).to.be.true;
           }
+
+          const isLovell700AlreadyActive = await Runners.common.forks.lovell700.isAlreadyActive();
+          if (isLovell700AlreadyActive) {
+            const pegoutTransactionCreatedEvent = await rskUtils.findEventInBlock(localRskTxHelper, 'pegout_transaction_created');
```

Comment:
Is it time to have these event names declared in an enum like structure? What do you think?